internal/commands/builder_create.go

```diff
@@ -87,6 +88,15 @@ Creating a custom builder allows you to control what buildpacks are used and wha
 				return err
 			}
 
+			multiArchCfg, err := processMultiArchitectureConfig(logger, flags.Targets, builderConfig.Targets, !flags.Publish)
+			if err != nil {
+				return err
+			}
+
+			if len(multiArchCfg.Targets()) == 0 {
+				logger.Warnf("A new '--target' flag is available to set the platform")
```

Comment:
Should we also mention `[[targets]]` in buildpack.toml?

Reply:
This warning is a bit weird to me. What if the user didn't intend to use targets?